wikipedia-sentences-incremental/app.py

```diff
@@ -0,0 +1,112 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+import itertools as it
+
+import click
+from jina.flow import Flow
+from jina import Document
```

Comment:
If we don't use Document and it, we could remove them. But if they will be used it's ok to keep them.

Reply:
Done